tools/manifest.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2015-2018 ACSONE SA/NV
+# License AGPLv3 (http://www.gnu.org/licenses/agpl-3.0-standalone.html)
+
+import ast
+import os
+
+MANIFEST_NAMES = ('__manifest__.py', '__openerp__.py', '__terp__.py')
+
+
+class NoManifestFound(Exception):
+    pass
+
+
+def get_manifest_path(addon_dir):
+    for manifest_name in MANIFEST_NAMES:
+        manifest_path = os.path.join(addon_dir, manifest_name)
+        if os.path.isfile(manifest_path):
+            return manifest_path
+
+
+def parse_manifest(s):
+    return ast.literal_eval(s)
```

Comment:
Why mature? I was proposing alpha as default, so maintainers can propose the level they consider appropriate with PR's and let the PSC approve that on a case by case basis. In any case we need to lay down the criteria before pushing this.

Reply:
Agree: default should be the lowest: the contributor has to learn the way up

Reply:
I'm sorry, the default should rather be "Production/Stable",
The existing policy is that merged modules are Stable.
So, all current modules should be displayed as Stable, and we don't need to do a mass change to all modules.
New modules to be merged as Alpha or Beta should carry that in their manifest. This could be CI checked.

Reply:
> The existing policy is that merged modules are Stable.

Let's finalize the criteria and then see if that holds true for all existing modules. In which case we can make a script to add it to existing modules. But it must not be the default, as the default would apply to all new modules.

Reply:
+1 @sbidoul 
I'm also in favor of a validation from the developer that his module his mature instead of the way around.

Thus keeping Alpha as default.

Reply:
@yvaucher thanks for your input. In case you have not seen it the criteria for the levels are being discussed in a [google doc](https://docs.google.com/document/d/1MmGxv0yTw0pvB-PYAcUhbCcku9ZG-Y2DSGXwZDNYyIw/edit#). In that document the current proposed default is Beta.